projects/plugins/jetpack/_inc/client/performance/media.jsx

```diff
@@ -82,6 +106,25 @@ class Media extends React.Component {
 				hideButton
 			>
 				{ foundVideoPress && videoPressSettings }
+				{ foundVideoPress && ! hasUpgrade && (
+					<JetpackBanner
+						className="media__videopress-upgrade"
+						callToAction={ __( 'Upgrade', 'jetpack' ) }
+						title={ sprintf(
+							/* translators: placeholder is a number. */
+							__(
+								'%d/1 free videos used. Upgrade now to unlock more videos and 1TB of storage.',
```

Comment:
Do we need an s here? The wording feels a bit odd:

![image](https://user-images.githubusercontent.com/426388/134153124-59d9f93b-ee21-4f7a-8b82-aa3a69664f12.png)

Maybe we don't need the `1` number?

Reply:
I'm good with that